integration/spark2/src/main/scala/org/apache/spark/sql/hive/CarbonMetaStore.scala

```diff
@@ -142,6 +142,10 @@ trait CarbonMetaStore {
 
   def getTableFromMetadataCache(database: String, tableName: String): Option[CarbonTable]
 
+  def getTableInfo(absoluteTableIdentifier: AbsoluteTableIdentifier)
```

Comment:
Why this is required? It is better not to add more interface

Reply:
many places we are reading schema from the tablePath, ie we decided to move it at common place.
Once this PR is merged will refactor code from other places to use this interface.

Reply:
This idea is good, but I think you do not need to put this in CarbonMetaStore trait, right? Reading schema from tablePath can be an utility.

Reply:
fixed added in schema redear